$NetBSD: patch-ad,v 1.3 2011/09/02 08:06:55 he Exp $

Don't try to include OS version in directory name for mib files.

--- tnm/generic/tnmInit.c.orig	1999-03-05 22:32:26.000000000 +0000
+++ tnm/generic/tnmInit.c
@@ -106,15 +106,12 @@ InitVars(interp)
 
     machine = Tcl_GetVar2(interp, "tcl_platform", "machine", TCL_GLOBAL_ONLY);
     os = Tcl_GetVar2(interp, "tcl_platform", "os", TCL_GLOBAL_ONLY);
-    vers = Tcl_GetVar2(interp, "tcl_platform", "osVersion", TCL_GLOBAL_ONLY);
 
     Tcl_DStringInit(&arch);
-    if (machine && os && vers) {
+    if (machine && os) {
 	Tcl_DStringAppend(&arch, machine, -1);
 	Tcl_DStringAppend(&arch, "-", 1);
 	Tcl_DStringAppend(&arch, os, -1);
-	Tcl_DStringAppend(&arch, "-", 1);
-	Tcl_DStringAppend(&arch, vers, -1);
     } else {
 	Tcl_DStringAppend(&arch, "unknown-os", -1);
     }
